### PR TITLE
Knowledge nuggets update

### PR DESCRIPTION
**Pull request by antooni**
Resolves L2B-330

### CODE REVIEW

File: packages/config/src/bridges/aptos.ts [hunk 1]
```diff
@@ -279,4 +279,11 @@ export const aptos: Bridge = {
       description: 'Participants of the 2/5 LayerZero MultiSig.',
     },
   ],
+  knowledgeNuggets: [
+    {
+      title: 'Security models analysis: isolated vs shared',
```
Comment (michalsidzej) [inline packages/config/src/bridges/aptos.ts:284]: <3

File: packages/config/src/bridges/cBridge.ts [hunk 2]
```diff
@@ -332,4 +333,17 @@ export const cBridge: Bridge = {
       ],
     },
   ],
+  knowledgeNuggets: [
+    {
+      title: 'cBridge deep dive',
+      url: 'https://twitter.com/bkiepuszewski/status/1437031523455229964?s=20',
+      thumbnailUrl: NUGGETS.BARTEK_TWITTER_THUMBNAIL
+    },
+
```
Comment (michalsidzej) [inline packages/config/src/bridges/cBridge.ts:342]:
```suggestion
    {
      title: 'How HTLC bridge works?',
      url: 'https://twitter.com/bkiepuszewski/status/1437031523455229964',
      thumbnailUrl: NUGGETS.BARTEK_TWITTER_THUMBNAIL
    },
```

File: packages/config/src/bridges/hop.ts [hunk 3]
```diff
@@ -203,4 +204,21 @@ export const hop: Bridge = {
...
+      title: 'How Hop protocol works?',
+      url: 'https://twitter.com/bkiepuszewski/status/1437320613358673922',
+      thumbnailUrl: NUGGETS.BARTEK_TWITTER_THUMBNAIL
+    },
+    {
+      title: 'How HTLC bridge works?',
+      url: 'https://twitter.com/bkiepuszewski/status/1437031523455229964',
+      thumbnailUrl: NUGGETS.BARTEK_TWITTER_THUMBNAIL
+    },
+  ],
```
Comment (michalsidzej) [inline packages/config/src/bridges/hop.ts:223]: Let's maybe agree on a consistant sort of the nuggets? I'm compraing it to cBridge here

File: packages/config/src/layer2s/common/nuggets.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+const STARKEX_NUGGETS: KnowledgeNugget[] = [
+  {
+    title: 'Understand StarkWare #1',
+    url: 'https://twitter.com/bkiepuszewski/status/1480473352213041152',
+    thumbnailUrl: BARTEK_TWITTER_THUMBNAIL
+  },
+  {
+    title: 'Understand StarkWare #2',
+    url: 'https://twitter.com/bkiepuszewski/status/1480804584926949377?s=20&t=vMgR4jW1ssap-A-MBsO4Jw',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/common/nuggets.ts:19]:
```suggestion
    url: 'https://twitter.com/bkiepuszewski/status/1480804584926949377',
```

File: packages/config/src/layer2s/common/nuggets.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    thumbnailUrl: BARTEK_TWITTER_THUMBNAIL
+  },
+  {
+    title: 'Understand StarkWare #3',
+    url: 'https://twitter.com/bkiepuszewski/status/1481173682219270146?s=20&t=vMgR4jW1ssap-A-MBsO4Jw',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/common/nuggets.ts:24]:
```suggestion
    url: 'https://twitter.com/bkiepuszewski/status/1481173682219270146',
```

File: packages/config/src/layer2s/common/nuggets.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    thumbnailUrl: BARTEK_TWITTER_THUMBNAIL
+  },
+  {
+    title: 'Understand StarkWare #4',
+    url: 'https://twitter.com/bkiepuszewski/status/1481521079562784769?s=20&t=vMgR4jW1ssap-A-MBsO4Jw',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/common/nuggets.ts:29]:
```suggestion
    url: 'https://twitter.com/bkiepuszewski/status/1481521079562784769',
```

File: packages/config/src/layer2s/dydx.ts [hunk 7]
```diff
@@ -203,4 +204,12 @@ export const dydx: Layer2 = {
         'dYdX V4 will be developed as a standalone blockchain based on the Cosmos SDK.',
     },
   ],
+  knowledgeNuggets: [
+    {
+      title: 'How does escape hatch work?',
+      url: 'https://twitter.com/bkiepuszewski/status/1469201939049103360?s=20&t=vMgR4jW1ssap-A-MBsO4Jw',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/dydx.ts:210]:
```suggestion
      url: 'https://twitter.com/bkiepuszewski/status/1469201939049103360',
```

File: packages/config/src/layer2s/bobanetwork.ts [hunk 8]
```diff
@@ -392,4 +393,11 @@ export const bobanetwork: Layer2 = {
       description: 'BOBA token launched by OMG Foundation.',
     },
   ],
+  knowledgeNuggets: [
+    {
+      title: 'What is Hybrid Compute?',
+      url: 'https://twitter.com/bkiepuszewski/status/1521849011594010624?s=20&t=vMgR4jW1ssap-A-MBsO4Jw',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/bobanetwork.ts:399]:
```suggestion
      url: 'https://twitter.com/bkiepuszewski/status/1521849011594010624',
```

File: packages/config/src/layer2s/arbitrum.ts [hunk 9]
```diff
@@ -505,4 +506,16 @@ export const arbitrum: Layer2 = {
...
   },
+  knowledgeNuggets: [
+    {
+      title: 'Arbitrum update increasing decentralization',
+      url: 'https://twitter.com/bkiepuszewski/status/1594754717330309120',
+      thumbnailUrl: NUGGETS.BARTEK_TWITTER_THUMBNAIL,
+    },
+    {
+      title: 'Arbitrum is down - but has it really stopped working?',
+      url: 'https://twitter.com/bkiepuszewski/status/1438445910191710211?s=20&t=vMgR4jW1ssap-A-MBsO4Jw',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/arbitrum.ts:517]:
```suggestion
      url: 'https://twitter.com/bkiepuszewski/status/1438445910191710211',
```

File: packages/config/src/bridges/stargate.ts [hunk 10]
```diff
@@ -271,12 +271,23 @@ export const stargate: Bridge = {
...
   ],
+  knowledgeNuggets: [
+    {
+      title: 'Security models analysis: isolated vs shared',
+      url: 'https://medium.com/l2beat/circumventing-layer-zero-5e9f652a5d3e',
+      thumbnailUrl: 'https://miro.medium.com/v2/resize:fit:720/format:webp/1*7iEM_gHFZ_Ws_r8N_DP20w.jpeg'
+    },
+    {
+      title: 'StarGate Bridge architecture',
+      url: 'https://twitter.com/bkiepuszewski/status/1518568490147450880?s=20&t=vMgR4jW1ssap-A-MBsO4Jw',
```
Comment (michalsidzej) [inline packages/config/src/bridges/stargate.ts:289]:
```suggestion
      url: 'https://twitter.com/bkiepuszewski/status/1518568490147450880',
```

File: packages/config/src/bridges/multichain.ts [hunk 11]
```diff
@@ -184,4 +185,16 @@ export const multichain: Bridge = {
...
     },
   ],
+  knowledgeNuggets: [
+    {
+      title: 'Multichain deep dive',
+      url: 'https://li.fi/knowledge-hub/multichain-a-deep-dive/',
+      thumbnailUrl: NUGGETS.LIFI_THUMBNAIL,
+    },
+    {
+      title: 'Multichain problem with escrow',
```
Comment (michalsidzej) [inline packages/config/src/bridges/multichain.ts:195]: Maybe:
```suggestion
      title: 'Multichain escrow problem',
```

File: packages/config/src/bridges/hop.ts (hunk 3 above)
Comment (antooni) [reply packages/config/src/bridges/hop.ts:223]: Let's postpone this conversation for the next iteration

**Review by michalsidzej — APPROVED** [review]
LGTM

**Pull request merged**